src/tests/D3MMom.t.sol

```diff
@@ -16,50 +16,43 @@
 
 pragma solidity ^0.8.14;
 
-import {DssTest} from "dss-test/DssTest.sol";
-import "./interfaces/interfaces.sol";
+import "dss-test/DssTest.sol";
 
 import {D3MMom} from "../D3MMom.sol";
 
-import {D3MTestPlan} from "./stubs/D3MTestPlan.sol";
-interface Hevm {
-    function warp(uint256) external;
-
-    function store(
-        address,
-        bytes32,
-        bytes32
-    ) external;
-
-    function load(address, bytes32) external view returns (bytes32);
+contract PlanMock {
+    bool public disabled;
+    function disable() external {
+        disabled = true;
+    }
 }
 
 contract D3MMomTest is DssTest {
-    D3MTestPlan d3mTestPlan;
-    D3MMom d3mMom;
+
+    PlanMock plan;
+    
+    D3MMom mom;
 
     function setUp() public {
-        d3mTestPlan = new D3MTestPlan(address(123));
+        plan = new PlanMock();
 
-        d3mTestPlan.file("maxBar_", type(uint256).max);
-        d3mTestPlan.file("bar", type(uint256).max);
-        d3mMom = new D3MMom();
-        d3mTestPlan.rely(address(d3mMom));
+        mom = new D3MMom();
     }
 
     function test_can_disable_plan_owner() public {
-        assertEq(d3mTestPlan.bar(), type(uint256).max);
+        assertEq(plan.disabled(), false);
 
-        d3mMom.disable(address(d3mTestPlan));
+        mom.disable(address(plan));
 
-        assertEq(d3mTestPlan.bar(), 0);
+        assertEq(plan.disabled(), true);
     }
 
     function test_disable_no_auth() public {
-        d3mMom.setOwner(address(0));
-        assertEq(d3mMom.authority(), address(0));
-        assertEq(d3mMom.owner(), address(0));
+        mom.setOwner(address(0));
+        assertEq(mom.authority(), address(0));
+        assertEq(mom.owner(), address(0));
 
-        assertRevert(address(d3mMom), abi.encodeWithSignature("disable(address)", address(d3mTestPlan)), "D3MMom/not-authorized");
+        assertRevert(address(mom), abi.encodeWithSignature("disable(address)", plan), "D3MMom/not-authorized");
```

Comment:
Should we start removing the `assertRevert` in favor the native Foundry `expectRevert`?